config/schedule.rb

```diff
@@ -7,9 +7,13 @@
   runner 'RecalculateAllLoanHealthJob.perform_later'
 end
 
+every 1.day, at: '2am' do
+  rake "madeline:enqueue_update_loans_task"
+end
+
```

Comment:
As far as I understand, `schedule.rb` just updates the crontab at deploy. If we want different frequency on staging versus production, perhaps we'd need to have different schedule.rb files per environment?

Reply:
According to @meltheadorable, not a blocking concern.